prime-router/docs/proposals/lineage/README.md

```diff
@@ -0,0 +1,123 @@
+- Similar for senders:  summary info on numbers of reports sent, number of items sent, and a list of senders to whom reports have been sent.
+- Find data that has "dead-ended" in the system, either because of a failure, or because it simply doesn't pass any of our jurisdictional filters, so it is never sent.
+
+### General Goals
+
+- Build trust in the Hub - the schema should enable features that give both receivers and senders a sense of trust in the system.  The hub should be able to reassure our users that their data has transferred safely and securely.
+- Secure data at all steps, both in-transit and at rest, while still making it available to those who are authorized access.
+
+### Future Use Cases
+
+For the first rounds of work, I'm avoiding tracking any specific data *internal* to a report, other than the `tracking_elements`.   However, the schema is designed to be expanded in future sprints, to potentially accomodate these use cases:
+
+- Tracking Positivity rates, sharp increases/drops for specific senders, geographic spikes, etc.
+
+## Definitions
+
+In the Hub, an **Item** is a single record of data.  A typical **Item** might be a Covid test result for one patient.   If the data is in CVS format, an **Item** is a single row in the CSV file.
+
+A **Report** in the hub is simply a set of **Item**, with additional metadata.   For example, if the data is in CSV format, the Hub expects a single row of Column Header strings, followed by one or more rows of **Item** data.    It is acceptable for a Report to have just a single Item.  A Report with no Items is not processed.
+
+The Hub implements a simple workflow engine.   Each step in the engine is a **Task**.  Most (but not all) tasks create one or more Reports. Current **Task** in the system include: 
+
+| Task      |     Description
+| ---       | ---
+| Receive   | Receive a new Report into the Hub
+| Transform | Validate a received report, and transform into desired output formats, according to configured data filter business rules
+| Batch     |  Merge one or more reports together.   Typically done to consolidate data per receiver requirements.
+| Send      | Reliably delivery data to an external receiver, eg, an API, sftp site, or provide for upload (and notify with an email)
+| Wipe      | Remove data, per business rules
+
+An **Organization** can recieve and/or send data to the hub.  A typical Organization is a state or local Public Health Department.   Strac, Redox, and SimpleReport are all also Organizations in the hub.
+
+An **OrganizationService** is an endpoint to which the Hub can send a specific kind of data to a specific place, so that the Organiation can use the data.  For example, an OrganizationService might be an sftp site where a local health department wants hl7 files uploaded every 4 hours.
+
```

Comment:
In my proposal to come, I'm thinking about going back to Senders and Receivers to OrganizationClient and OrganizationService. Perhaps, if you agree, then you can make the same change.

Reply:
At this point, I'm ok either way.    Both require care - its easy to wrap yourself in linguistic spirals.  Like, what do you call the results of a 'send' from the Hub to a PHD?   Its the "received_result", true?    Which sounds like the hub is receiving something....sigh.